examples/java/spring-boot/src/main/java/uk/gov/api/springboot/application/services/ApiService.java

```diff
@@ -1,20 +1,13 @@
 package uk.gov.api.springboot.application.services;
 
-import java.util.List;
 import org.springframework.stereotype.Service;
-import uk.gov.api.springboot.domain.model.Api;
 import uk.gov.api.springboot.domain.model.repositories.ApiStorage;
+import uk.gov.api.springboot.domain.services.Provider;
 
 @Service
-public class ApiService {
-
-  private final ApiStorage storage;
+public class ApiService extends Provider {
```

Comment:
We should have an `ApiServiceTest`, too, which for now can be a duplicate of `ProviderTest`, but in the future would include other things